teacher.rb

```diff
@@ -0,0 +1,17 @@
+require './person'
+
+class Teacher < Person
+  attr_reader :specialization
+
+  def initialize(specialization, age, name = 'unknown', parent_permission: true)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+    super
+    @specialization = specialization
```

Comment:
- [ ] Kindly use your super to send your parameters to the parent class as shown below 👍 
 
⬇️
`super(@age, @name, @parent_permission)`